admin_guide/multiproject_quota.adoc

```diff
@@ -0,0 +1,100 @@
+[[admin-guide-muliproject-quota]]
+= Multi-Project Quotas
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
+to be shared across multiple projects.  Resources used in each selected project
+will be aggregated and that aggregate will be used to limit resources across
+all the selected projects.
+
+=== Selecting Projects
+
+Projects can be selected based on either annotation selection, label selection, or both.
+For example, `oc create clusterquota for-user --project-annotation-selector openshift.io/requester=deads --hard pods=10 --hard secrets=20`
```

Comment:
We tend to use generic user names, so I am going to unfortunately be a killjoy and recommend replacing instances of "deads" with a replaceable <user-name>  :crying_cat_face: